images/shared_files/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+useradd -l -g "$NAEMON_GID" -u "$NAEMON_UID" -s /bin/bash -M naemon -d /var/lib/naemon/
+
+curl https://download.opensuse.org/repositories/home:/itrs-op5/CentOS_7/home:itrs-op5.repo -o /etc/yum.repos.d/merlin.repo
+curl https://download.opensuse.org/repositories/home:/naemon/CentOS_7/home:naemon.repo -o /etc/yum.repos.d/naemon.repo
+yum install -y naemon-core naemon-livestatus merlin-slim monitor-merlin-slim merlin-apps-slim && \
+  yum clean all && \
+  rm -rf /var/cache/yum /etc/yum.repos.d/op5-release.repo
```

Comment:
Do we ever install the op5-release package?. If not I think the /etc/yum.repos.d/op5-release.repo could be removed